<!-- -*- mode: XML -*- -->
<!-- run ibeo_had_node for Ibeo Lux/Scala lidar with ECU -->

<launch>

  <!-- declare arguments with default values -->
  <arg name="ibeo_had_ip" default="192.168.100.211"/>
  <arg name="ibeo_had_port" default="12002"/>
  <arg name="rate" default="20" />

  <!-- node for HAD -->
  <node pkg="ibeo_had" type="ibeo_had_ecu" name="ibeo_listener">
    <param name="ibeo_had_ip" value="$(arg ibeo_had_ip)" />
    <param name="ibeo_had_port" value="$(arg ibeo_had_port)" />
    <param name="rate" value="$(arg rate)" />
  </node>

</launch>
